historic_cadastre/models.py

```diff
@@ -93,9 +93,24 @@ class AuthenticationUserRole(Base):
     )
 
 
-mapped_classes_registry = {}
+class HistoricParcelTree(Base):
+    __tablename__ ='historic_parcel_tree'
+    __table_args__ = {'schema': 'plan_histo', 'autoload': True}
+    idobj = Column('idobj', String, primary_key=True)
+    imm_dest = Column(String, ForeignKey('plan_histo.historic_parcel_tree.imm_source'))
+    children = relationship("HistoricParcelTree", lazy="joined", join_depth=6)
+
+
 for value in globals().values():
     if hasattr(value, '__tablename__') and value.__tablename__.startswith('authentication_') is False:
         if hasattr(value, '__table_args__'):
             mapped_class = value
             mapped_classes_registry[mapped_class.__tablename__] = mapped_class
+    mapped_classes_registry = {}
```

Comment:
Cette ligne doit se trouver juste avant la 104 `for value in globals().values():`